app/helpers/application_helper.rb

```diff
@@ -153,36 +153,44 @@ def available_courses
       {
         badge_image_url: 'badge-html-css.svg',
         badge_alt_text: 'html css badge',
-        title: 'HTML + CSS'
+        title: 'HTML + CSS',
+        title_url: '/courses/html5-and-css3'
```

Comment:
`path` would be a more appropriate name for this as its not a full url. What do you think?

Reply:
Sure i will do that